action/audit.go

```diff
@@ -122,7 +123,7 @@ func (s *Action) audit(validator *crunchy.Validator, secrets <-chan string, chec
 	for secret := range secrets {
 		content, err := s.Store.GetFirstLine(secret)
 		if err != nil {
-			checked <- auditedSecret{name: secret, content: string(content), err: err}
+			checked <- auditedSecret{name: secret, content: string(content), err: err, message: err.Error()}
 			continue
 		}
 
```

Comment:
I'm adding messages for errors, so they show up in the output, grouped like the other messages, e.g. "Failed to decrypt".

Which is indeed an error in the pipeline, but that's worth highlighting in an audit. If you dislike warnings and errors being mixed together in the output, we could separate the two groups or color-highlight them.